behind-the-veil-siteroot/imports/ui/components/util.jsx

```diff
@@ -19,10 +19,11 @@ export function getUserInfo() {
     const [userInfo, setUserInfo] = useState(
```

Comment:
Just a minor thing, if this util needs to create a state for userInfo (i.e. call useState), then this function is a custom react hook. I suggest renaming the function to something like 'useUserInfo' to convey that it's a hook

Reply:
yup! sounds good